packages/e2e-cypress/src/helpers/cct-dev-server/index.ts

```diff
@@ -43,12 +82,27 @@ async function quasarSharedConfig(
   };
 }
 
-async function quasarWebpackConfig(quasarAppPackage: string) {
+async function getQuasarConfig(
```

Comment:
This fuction doesn't seem to do anything except than removing `quasarAppPackage` from the returned object
Can we simplify and remove it, using `quasarSharedConfig` directly?